test/src/common/js_proxy_test.dart

```diff
@@ -68,6 +75,15 @@ main() async {
       expect(model.finalVal, 1);
     });
 
+    test('getter/setter fields only need to annotate the getter', () {
+      expect(model.getterSetterVal, 1);
+      expect(model.jsProxy['getterSetterVal'], model.getterSetterVal);
```

Comment:
nit - maybe just do:
```dart
expect(model.jsProxy['getterSetterVal'], 1);
```

not that it would, but it makes it clear that the value is the same and it doesn't change. Alternatively:
```dart
var value = model.getterSetterVal;
expect(value, 1)
expect(model.jsProxy['getterSetterVal'], value);
```